packages/store/src/public_api.ts

```diff
@@ -46,3 +46,7 @@ export { NgxsExecutionStrategy } from './execution/symbols';
 export { ActionType, ActionOptions } from './actions/symbols';
 export { NoopNgxsExecutionStrategy } from './execution/noop-ngxs-execution-strategy';
 export { StateToken } from './state-token/state-token';
+export {
+  UnhandledActionsLogger,
+  ignoreActions
```

Comment:
This `ignoreActions` API feels like it doesn't match the rest of the public API of the library.